docker/install/conda_env/tensorflow_gpu.yml

```diff
@@ -4,7 +4,6 @@ dependencies:
   - pip
   - pip:
     - tensorflow==2.3.0
-    - tfdlpack-gpu
```

Comment:
why remove this line? this package is no longer needed any more?

Reply:
I think you'd better do not change files under //docker as the modifications you made will not take effect directly. Images used in CI remains to be the previously built ones.